packages/auth-providers/azureActiveDirectory/setup/src/templates/web/auth.ts.template

```diff
@@ -4,7 +4,7 @@ import { createAuth } from '@redwoodjs/auth-azure-active-directory-web'
 
 const azureActiveDirectoryClient = new PublicClientApplication({
   auth: {
-    clientId: process.env.AZURE_ACTIVE_DIRECTORY_CLIENT_ID || '',
```

Comment:
Have you checked if this makes a difference in TS strict mode?

Reply:
No have not, just struck me as not making sense from a JS perspective, but I'll check

Reply:
I'll just revert till I have time to test